sessions/recap-project-5/example-implementation/pages/index.js

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+import Header from "../components/Header";
+
+const ImageContainer = styled.div`
+  position: relative;
+  width: 600px;
+  height: 100%;
+`;
+
+const StyledImage = styled(Image)`
+  object-fit: cover;
+`;
+
+export default function Spotlight({ pieces }) {
+  const randomPiece = pieces[Math.floor(Math.random() * pieces.length)];
+
+  return (
+    <>
```

Comment:
User Story 2 speaks about three props for the `Spotlight` component (image, title, author). The ladder two are not yet displayed (but I do not have to latest wireframes to check how it should look like).

Reply:
In my personal opinion, displaying author and image is enough. 🤷‍♀️ Otherwise I'd probably handle the design differently.

Reply:
Let's add the author here in the `Header` component right now there is static text `ARTIST`:

components/Header.js

```suggestion
<Spotlight>
        Spotlight: <small>©</small>ARTIST
</Spotlight>
```

Not having the `title` is okay